### PR TITLE
daisyui implemented 

### PR DESCRIPTION
**Pull request by cpwallah**
hi bro added the changes which you asked for in the required branch

### CODE REVIEW

File: frontend/app/globals.css [hunk 1]
```diff
@@ -2,26 +2,4 @@
...
-body {
-  color: rgb(var(--foreground-rgb));
-  background: linear-gradient(
-      to bottom,
-      transparent,
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
-}
+@import 'daisyui/dist/full.css';
```
Comment (lokytech5) [inline frontend/app/globals.css:5]: Remove this import statement from the global CSS file.

File: frontend/app/globals.css
Comment (lokytech5) [inline frontend/app/globals.css:1]: Remove this import statement from the global CSS file.

**Review by lokytech5 — CHANGES_REQUESTED** [review]
Remove the daisyui import statement added in the global CSS for a quick merge

**Comment by lokytech5** [comment]
Thank you very much for contributing. Please complete the task so i can merged. A couple of good first issue we be available today.

File: frontend/app/globals.css
Comment (cpwallah) [reply frontend/app/globals.css:1]: @lokytech5 done it

**Review by lokytech5 — APPROVED** [review]
Well done bro. more first issue coming up?

**Pull request merged**